### PR TITLE
UPP Lore - UPP Assault Pioneers

### PR DESCRIPTION
**Pull request by Misttyy**
## About The Pull Request

This PR changes a lot of small naming conventions and the sort for the UPP. This is something I've discussed with fellow lore contributors Monkey and Lago. UPP Ranks now follow 1980s USSR ranks during the Soviet-Afghan war, with the Latinization removed and simply converted to English.

Additionally, UPP Kommandos have been renamed to UPP Assault Pioneers as apart of a lore project. 

## Why It's Good For The Game

This sets the path for a feature UPP ERT to be added (non covert UPP Assault Pioneers). As well, this makes UPP ranks look far less silly and makes everything comply with the lore more.

## Changelog
:cl: Mistfrag
del: Removed UPP conscripts from the code.
spellcheck: Converts UPP Ranks to follow USSR ranks without the silly looking Latinization. Renames UPP Kommandos to UPP Assault Pioneers (Covert).
spellcheck: Fixes some instances of Tychon's Rift in ERT desc.
:cl:

### CODE REVIEW

**Comment by stanalbatross** [comment]
UPP ERTs no longer default spawn them hostile, the UPP are not at war with the UA, this did not make sense. However, UPP Covert ERT still spawns hostile with specified instructions. 

pretty big thing not documented in cl

**Comment by Misttyy** [comment]
> UPP ERTs no longer default spawn them hostile, the UPP are not at war with the UA, this did not make sense. However, UPP Covert ERT still spawns hostile with specified instructions. 
> 
> 
> 
> pretty big thing not documented in cl

I'm going to remove this tonight and wait to do it in another PR. I think, while a good change, it needs more framework to support it.

**Review by Geevies — CHANGES_REQUESTED** [review]
i'd say keep the UPP conscripts in, they're useful (and fun!) for weak event characters

**Comment by Misttyy** [comment]
Fixed up merge conflicts, readded conscripts as an event role.

File: code/game/machinery/computer/demo_sim.dm [hunk 1]
```diff
@@ -185,8 +185,6 @@
 			switch(dummy_mode)
 				if(CLF_MODE)
 					user.client.cmd_admin_dress_human(dummy, "CLF Soldier", no_logs = TRUE)
-				if(UPP_MODE)
```
Comment (carlarctg) [inline code/game/machinery/computer/demo_sim.dm:188]: why did you remove this

File: code/game/objects/effects/landmarks/freed_mob_spawner.dm [hunk 2]
```diff
@@ -17,11 +17,6 @@
 	arm_equipment(H, equipment_path, TRUE, count_participant)
 	H.free_for_ghosts()
 
-/obj/effect/landmark/freed_mob_spawner/upp_conscript
```
Comment (carlarctg) [inline code/game/objects/effects/landmarks/freed_mob_spawner.dm:20]: why did you remove this

File: code/game/objects/items/storage/backpack.dm [hunk 3]
```diff
@@ -922,8 +922,8 @@ GLOBAL_LIST_EMPTY_TYPED(radio_packs, /obj/item/storage/backpack/marine/satchel/r
 
 /obj/item/storage/backpack/marine/satchel/scout_cloak/upp
 	name = "\improper V86 Thermal Cloak"
-	desc = "A thermo-optic camouflage cloak commonly used by UPP commando units."
-	uniform_restricted = list(/obj/item/clothing/suit/storage/marine/faction/UPP/commando) //Need to wear UPP commando armor to equip this.
```
Comment (carlarctg) [inline code/game/objects/items/storage/backpack.dm:926]: i mean it still says commando in the subtype.................................

File: code/modules/gear_presets/upp.dm [hunk 4]
```diff
@@ -417,10 +417,10 @@
 	assignment = JOB_UPP_ENGI
 	rank = JOB_UPP_ENGI
 	role_comm_title = "Sap"
-	paygrade = "UE3S"
+	paygrade = "UE3"
 
 /datum/equipment_preset/upp/sapper/load_gear(mob/living/carbon/human/H)
-	//Sappers should have lots of gear and whatnot that helps them attack or siege marines
```
Comment (carlarctg) [inline code/modules/gear_presets/upp.dm:423]: whye

**Review by harryob — CHANGES_REQUESTED** [review]
this is gonna stale if you don’t resolve carlarc’s threads

**Pull request closed without merging**